config/config.exs

```diff
@@ -223,6 +223,11 @@ config :appsignal, :config,
     "Unlock.Controller#fetch"
   ]
 
+config :transport, Transport.Mailer,
+  adapter: Swoosh.Adapters.Mailjet,
+  api_key: System.get_env("MJ_APIKEY_PUBLIC"),
+  secret: System.get_env("MJ_APIKEY_PRIVATE")
+
```

Comment:
Je serais plutôt d'avis de mettre ça dans `runtime.exs`, et uniquement sur les environnements qui nous intéressent (production oui, staging non, les clés maillets étant non configurées là bas par ailleurs).

Reply:
Merci je vais aller voir ! Pour préciser ma motivation aussi, c'est que je vise à terme à ce que notre code-base soit compatible avec le fait de générer une "release" Elixir (c'est une version totalement compilée de l'application, voir https://elixir-lang.org/getting-started/mix-otp/config-and-releases.html), sur une machine de build (ou un GitHub action) qui n'aurait pas la configuration "sensible" de production (configuration sensible qui serait uniquement lue au runtime, au moment où les credentials sont disponibles).

C'est un schéma qui permettra d'aller héberger ailleurs si on le souhaite un peu plus facilement (dans Docker ou sans Docker).

Voilà pour la petite histoire :smile: